apps/web/server/routers/viewer/slots.tsx

```diff
@@ -151,16 +151,17 @@ export const slotsRouter = createRouter().query("getSchedule", {
     const startTime =
       input.timeZone === "Etc/GMT"
         ? dayjs.utc(input.startTime)
-        : dayjs.utc(input.startTime).tz(input.timeZone, true);
+        : // The time is already sent in UTC, so mark it UTC by passing true flag and without changing the time. After that convert into the timezone
+          dayjs(input.startTime).utc(true).tz(input.timeZone);
     const endTime =
       input.timeZone === "Etc/GMT"
         ? dayjs.utc(input.endTime)
-        : dayjs.utc(input.endTime).tz(input.timeZone, true);
+        : // The time is already sent in UTC, so mark it UTC by passing true flag and without changing the time. After that convert into the timezone
+          dayjs(input.endTime).utc(true).tz(input.timeZone);
```

Comment:
```suggestion
          dayjs(input.endTime).utc().tz(input.timeZone);
```

Because it's hard to explain why that's needed; see the following, the problem occurs when system time != UTC:
<img width="882" alt="image" src="https://user-images.githubusercontent.com/1046695/177543414-dc4366a9-1653-4804-b407-d4976faa1134.png">

Reply:
@emrysal should we open this issue in the days repo?